draft-ietf-quic-qpack.md

```diff
@@ -1004,6 +1002,119 @@ Description:
+| 22    | :scheme                          | http                                                  |
+| 23    | :scheme                          | https                                                 |
+| 24    | :status                          | 200                                                   |
+| 25    | :status                          | 304                                                   |
+| 26    | accept                           | */*                                                   |
+| 27    | accept                           | application/dns-message                               |
+| 28    | accept-encoding                  | gzip, deflate, br                                     |
+| 29    | accept-language                  |                                                       |
+| 30    | authorization                    |                                                       |
+| 31    | cache-control                    | max-age=0                                             |
+| 32    | cache-control                    | max-age=604800                                        |
+| 33    | cache-control                    | max-age=2592000                                       |
+| 34    | cache-control                    | no-cache                                              |
+| 35    | cache-control                    | no-cache, no-store, must-revalidate                   |
+| 36    | cache-control                    | public, max-age=31536000                              |
+| 37    | content-encoding                 | br                                                    |
+| 38    | content-type                     | application/dns-message                               |
+| 39    | content-type                     | application/javascript                                |
+| 40    | content-type                     | application/json                                      |
+| 41    | content-type                     | application/json; charset=utf-8                       |
+| 42    | content-type                     | application/x-www-form-urlencoded                     |
+| 43    | content-type                     | application/x-www-form-urlencoded; charset=utf-8      |
+| 44    | content-type                     | image/gif                                             |
+| 45    | content-type                     | image/jpeg                                            |
+| 46    | content-type                     | image/png                                             |
+| 47    | content-type                     | text/css                                              |
+| 48    | content-type                     | text/html; charset=utf-8                              |
+| 49    | content-type                     | text/javascript                                       |
+| 50    | content-type                     | text/javascript; charset=utf-8                        |
```

Comment:
https://datatracker.ietf.org/doc/draft-ietf-dispatch-javascript-mjs/ deprecates this, so remove both text/javascript options.